cmd/nginx-ingress/main.go

```diff
@@ -242,13 +242,14 @@ func main() {
 	var registry *prometheus.Registry
 	var managerCollector collectors.ManagerCollector
 	var controllerCollector collectors.ControllerCollector
+	var ingressClassLabel = map[string]string{"class": *ingressClass}
```

Comment:
style suggestion: for variable initialization, we use the following style:
```
ingressClassLabel := map[string]string{"class": *ingressClass}
```
Please change for consistency 

Additionally, since you're using a map, better to rename the variable to `constLabels` or something like that. If you want to use `ingressClassLabel`, you can safely change the type from map[string]string to string. String is enough to represent the value of one label.